user-guide/Troubleshooting/Known_issues/KI_Inter-DMA-connection_issues_when_ExternAuthentication_is_used.md

```diff
@@ -0,0 +1,39 @@
+---
+uid: KI_Inter-DMA-connection_issues_when_ExternAuthentication_is_used
+---
+
+# Inter-DMA connection issues when ExternalAuthentication is used
+
+## Affected versions
+
+From DataMiner 10.4.3 onwards.
```

Comment:
This issue was first reported in DataMiner version 10.2.8, so it will most likely also be present in the Main Release track. Adjusting the affected versions accordingly: 
```suggestion
From DataMiner 10.1.0 [CU17]/10.2.0 [CU5]/10.2.8 onwards.
```